src/App.js

```diff
@@ -2,13 +2,22 @@ import React, { Component } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import { FindMovie } from './components/FindMovie';
+
 import data from './api/movies.json';
 
 export class App extends Component {
   state = {
     movies: data,
   };
 
+  addMovie = (addedMovie) => {
+    if (!this.state.movies.find(movie => movie.imdbId === addedMovie.imdbId)) {
```

Comment:
is better to create some variable for that 'shouldAddMovie' or 'isAlreadyInList'